tests/testthat/test-derive_var_trtemfl.R

```diff
@@ -204,3 +204,14 @@ test_that("derive_var_trtemfl Test 9: error if `intensity` without `initial_inte
     "`initial_intensity` argument is ignored when `group_var` is specified"
   )
 })
+
+## Test 11: error if `group_var` are specified without `subject_keys` ----
+test_that("derive_var_trtemfl Test 11: error if `group_var` without `subject_keys`", {
+  expect_error(
+    derive_var_trtemfl(
+      adae,
+      group_var = AEGRPID,
+      subject_keys = NULL
+    )
+  )
```

Comment:
Please use `expect_snapshot()` to ensure that the correct error message is issued.

By the way, unit test for errors and warnings should never accept _any_ error or warning but always check that the expected one is issued (by using `expect_snapshot()` or the `class` argument of `expect_error()`/`expect_warning()` ).

@bms63 , maybe we should add this to the unit test guidance. What do you think?

Reply:
yes good idea we should update!! - i think i cleared an update for tests in derive_vars_transposed() that didn't meet our practices.  @jimrothstein no problems with the last PR that was on me I should of remembered this!!

but now we can document it...maybe it should even go in the PR template?  Hard to remember all these details!!

Reply:
> i think i cleared an update for tests in derive_vars_transposed() that didn't meet our practices.

I've fixed it in https://github.com/pharmaverse/admiral/pull/2592.

Reply:
what is your daily caffeine, food intake, and general regimen!!??  how are you so productive!!  thanks!!!

Reply:
Thank you @bundfussr.

Made note to myself and will look over your other suggestions.

> Please use `expect_snapshot()` to ensure that the correct error message is issued.
>

Reply:
> what is your daily caffeine, food intake, and general regimen!!?? how are you so productive!! thanks!!!

no coffee, just tea 🍵